storage/coop/rchain/key-value-store/KeyLexer.scala

```diff
@@ -95,30 +95,24 @@ class KeyLexer(lineIn: String) {
     var endOfString = false
 
     while (i < line.length && c != '(' && c != ')' && c != ',') {
-      c match {
-        case _ if Character.isLetter(c) => {
-          if (isDigits)
-            throw new Exception(
-              "KeyLexer.NextToken(): malformed (4): '"
-                + line + "', " + i)
-          isLetters = true
-          tokenStr ++= line(i).toString
-        }
-
-        case _ if Character.isDigit(c) => {
-          if (isLetters)
-            throw new Exception(
-              "KeyLexer.NextToken(): malformed (5): '"
-                + line + "', " + i)
-          isDigits = true
-          tokenStr ++= line(i).toString
-        }
-
-        case _ =>
+      if (Character.isLetter(c)) {
+        if (isDigits)
```

Comment:
In C/C++ it's not stylish to leave the {}'s off an if condition, because when someone comes around later to add additional statements, it's too easy to create bugs. Is there something about Scala that makes this okay?